rendercv/renderer/templater.py

```diff
@@ -119,11 +119,7 @@ def render_templates(self) -> tuple[str, str, list[tuple[str, list[str], str]]]:
         """
         # Template the preamble, header, and sections:
         preamble = self.template("Preamble")
-        header = (
-            self.template("HeaderWithPhoto")
-            if self.cv.photo
-            else self.template("Header")
-        )
+        header = self.template("Header")
```

Comment:
At the begginning I tought to split in 2 different templates for header with photo and header without photo, but after seeing that in moderncv it should be implemented in Preamble i reverted it. Also it wasn't a flexible solution, because I was putting as a constraint that the photo had to go in Header

Reply:
Yes, one `Header.j2.tex` file is the way to go; otherwise, there will be code duplication, and we will need to maintain them to be the same.